src/libLookup/Lookup.cpp

```diff
@@ -5986,12 +5883,6 @@ void Lookup::FetchMbTxPendingTxMessageFromL2l(uint64_t blockNum) {
           }
         }
 
-        // for each shard, send request for pending txn message to l2l data
```

Comment:
Can rename `FetchMbTxPendingTxMessageFromL2l` to `FetchMBnForwardTxMessageFromL2l`?